templates/templated_email/source/confirm_note.mjml

```diff
@@ -14,7 +14,7 @@
         </mj-text>
         <mj-text>
           {% blocktrans trimmed context "Note confirmation e-mail text" %}
-            A new note has been added to your order. To see details please visit: <a href="{{ url }}">{{ url }}</a>
+            A new note has been added to your order. To see details please visit: <a href="{{ order_details_url }}">{{ order_details_url }}</a>
```

Comment:
The old `url` context variable is also used in the plain-text version of this email in file "confirm_note.email", we should change it there as well. Except for that, I think we have covered all of the URLs.